.github/label.yml

```diff
@@ -0,0 +1,13 @@
+name: "PR Labeler"
+
+on:
+  pull_request:
+    branches: [ develop ]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/labeler@main
+        with:
+          repo-token: "${{ secrets.POLKADEX }}"
```

Comment:
@Gauthamastro insted of POLKADEX use secrets.GITHUB_TOKEN

Reply:
it's pre-defined

Reply:
But GITHUB_TOKEN is not stored in the secrets. Where will it get value then?

Reply:
It's still failing